### PR TITLE
Create Destination.cpp

### PR DESCRIPTION
**Pull request by AhmedHamdyKotp**
خالد هيظبط الكلاسين بتوعه وقشطا على كده

### CODE REVIEW

File: src/Destination.cpp [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+        Date startDate = activity.getStartDate();
+        Date endDate = activity.getEndDate();
+
+        activity.setName(name);
+        activity.setDescription(description);
+        activity.setLocations(locations);
+        activity.setPrice(price);
+        activity.setStartDate(startDate);
+        activity.setEndDate(endDate);
+    }
```
Comment (OsamaAshraf01) [inline src/Destination.cpp]: Why are you doing this step?

File: src/Destination.cpp [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+        vector<string> amenities = hotel.getAmenities();
+        double pricing = hotel.getPricing();
+
+        hotel.setName(name);
+        hotel.setLocation(location);
+        hotel.setType(type);
+        hotel.setDescription(description);
+        hotel.setAmenities(amenities);
+        hotel.setPricing(pricing);
+    }
```
Comment (OsamaAshraf01) [inline src/Destination.cpp]: Same here

File: src/Destination.cpp [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+        string availableFlights = airline.getAvailableFlights();
+        string availableAirports = airline.getAvailableAirports();
+        string supportedCountries = airline.getSupportedCountries();
+
+        airline.setName(name);
+        airline.setMaxWeight(maxWeight);
+        airline.setAvailableFlights(availableFlights);
+        airline.setAvailableAirports(availableAirports);
+        airline.setSupportedCountries(supportedCountries);
+    }
```
Comment (OsamaAshraf01) [inline src/Destination.cpp]: and here

**Review by OsamaAshraf01 — CHANGES_REQUESTED** [review]

File: src/Destination.cpp (hunk 1 above)
Comment (AhmedHamdyKotp) [reply src/Destination.cpp]: To give him the ability to assign his activities for this distension

Comment (OsamaAshraf01) [reply src/Destination.cpp]:
```C++
this->activities = activities;
```
Is enough

Comment (AhmedHamdyKotp) [reply src/Destination.cpp]: بحب اكتب يسطا  : )

Comment (OsamaAshraf01) [reply src/Destination.cpp]: The problem is although you wrote 15 redundant lines, those lines are doing nothing 😄.
you are getting the name of an activity and setting it in the same activity.
```C++
string name = activity.getName();
activity.setName(name);
```

**Pull request merged**